src/rewrite.rs

```diff
@@ -23,6 +25,21 @@ impl FunctionRewrite for JsonFunctionRewriter {
                     }
                 }
             }
+        } else if let Expr::BinaryExpr(bin_expr) = &expr {
+            if let Operator::Custom(WrapCustomOperator(op)) = &bin_expr.op {
+                if let Ok(json_op) = JsonOperator::try_from(op.name()) {
+                    let func = match json_op {
+                        JsonOperator::Arrow => crate::json_get_str::json_get_str_udf(),
+                        JsonOperator::LongArrow => crate::json_get::json_get_udf(),
```

Comment:
```suggestion
                        JsonOperator::Arrow => crate::json_get_str::json_get_udf(),
                        JsonOperator::LongArrow => crate::json_get::json_get_str_udf(),
```
I believe these are backwards relative to postgres 
<img width="473" alt="image" src="https://github.com/datafusion-contrib/datafusion-functions-json/assets/35119617/d54439a6-8e8f-4c67-922c-64126ffc3042">

https://www.postgresql.org/docs/9.5/functions-json.html

Reply:
very good spot, fixed.